### DISCUSSION TITLE
How do I unalias `co` from `branchless checkout`

### DISCUSSION THREAD

**Discussion by KevinWuWon** [arxanas/git-branchless#218] [Q&A] [answered]
When I type `git co`, it's triggering `git branchless checkout`. How do I make it not do that?

I can't find any config doing this in my `~/.gitconfig` or `.git/config` files. I actually have it aliased to plain `checkout` according to those configs.

I'm using the latest `git-branchless` from master.

If you want to know why I don't want I don't want to use `git branchless checkout`, it's because I want `git checkout green` to just go ahead and put me on the `green` branch, rather than showing me a UI to select which commit among all my commits with a partial substring match.

**Comment by arxanas** [comment, marked as answer]
Hi, don't have a computer right now, but check the file at `.git/branchless/config`.

I wasn't expecting it to override your normal config, so I can look into this in more detail later.

It might be useful to automatically check out `$1` if it's a branch name.

**Reply by KevinWuWon** [reply]
Thanks, that was it.

BTW the other reason is that I was using `git co -b branchname` to create branches.

**Comment by arxanas** [comment]
I filed #219 to track this issue.
